frontend/src/components/footer.tsx

```diff
@@ -15,28 +17,78 @@ import RiceLogoURL from '../images/rice-logo.png'
 import OpenStaxURL from '../images/openstax-logo.png'
 
 const Funders = () => {
+    const isMobile = useMediaQuery('(max-width: 768px)');
+    const isTablet = useMediaQuery('(max-width: 1024px)');
+
     return (
-        <Container bg='white'>
-            <Stack>
-                <Title order={4} className="fw-bold">Support from Scientific Agencies</Title>
-                <Flex justify='center' align='center' mx='auto'>
-                    <Anchor target="_blank" href="https://ies.ed.gov/">
-                        <Image alt="Institute of Education Sciences logo" src={IESLogoURL}/>
+        <Container bg="white" px={isMobile ? 16 : 32} pt={16} pb={40}>
+            <Stack gap={isMobile ? 16 : 24}>
```

Comment:
gap can take the following values: `xs, sm, md, lg, xl` Have a look [here](https://mantine.dev/styles/style-props/)